code-challenges.js

```diff
@@ -7,48 +7,116 @@
 // --------------------1) Create a function that takes in a number and determines if a temperature is below boiling point, at boiling point or above boiling point. Boiling point is 212 degrees Fahrenheit.
 // Use the test variables provided below. Expected output: "35 is below boiling point", "350 is above boiling point", "212 is at boiling point"
 
-var temp1 = 35
-var temp2 = 350
-var temp3 = 212
-
+//1. Start out with a declaration, us "const" because this will not be reassigned.
+//2. Begin with camelCased variable name. I will use boilingPoint as my variable name.
+//3. My variable must have an equal sign after it.
+//4. Now my I must have parentheses to describe an action and to pass inputs.
+//5. I need a function syntax "=>".
+//6. I need curly brackets to encapsolate my code "{}".
+//7. Finally, I need a return function.
+
+// for this problem:
+//-I need to create a variable that takes in a number to determine if a temperature is above, at, or below boiling point.
+
+// var temp1 = 35
+// var temp2 = 350
+// var temp3 = 212
+
+// var temp = ()
+//
+//   // if temp1 is above 212
+//   if(temp > 212) {
+//     return "is above boiling point"
+//   }
+// //   // if boilingPoint is equal to 212 return "is at boiling point"
+//   else if(temp === 212) {
+//     return "is at boiling point"
+//   }
+//    else if(temp < 212) {
+//     return "is below boiling point"
+//   }
+// }
+// console.log(boilingPoint(200))
 
 
 // --------------------2) Create a function that takes in two arrays of numbers and returns a single array with all the values sorted least to greatest.
 // Use the test variables provided below. Expected output: [-9, -7, 0, 3, 7, 8, 9, 13, 22, 36]
 
-var myNumbers1 = [3, 7, 0, 36, -9]
-var myNumbers2 = [8, -7, 22, 9, 13]
-
-
+// for this question I must create a function that returns a single array.
+// I must use the ".sort()" function to list the numbers from least to greatest
+// the first thing I need to do is to create the function
+// make my declaration
+// function name
+// equals sign
+// concatenate the two variables
+// return a value that orders the two arrays from least to greatest
+
+// var myNumbers1 = [3, 7, 0, 36, -9]
+// var myNumbers2 = [ , 8, -7, 22, 9, 13]
+//
+// var singleSortedArray = (myNumbers1 + myNumbers2) => {
+//   return singleSortedArray.sort()
+// }
+// console.log(singleSortedArray)
 
 // --------------------3) Create a function that takes in a string of a single word and returns the string with all letters reversed.
 // Use the test variables provided below. Expected output: "ovarb", "eilrahc"
 
-var myString1 = "bravo"
-var myString2 = "charlie"
-
+//  create a constant variable
+// make the function
+// add the equals sign
+// use the reversed function to render the two strings reversed order ".reversed"
 
+// var myString1 = "bravo"
+// var myString2 = " charlie"
+//
+// const singleWord = (myString1 + myString2) => {
+//   return singleWord.reversed
+// }
+//
+// console.log(singleWord)
```

Comment:
Really close! I would like to know what kind of results this code gave you and how you tried to trouble shoot it!